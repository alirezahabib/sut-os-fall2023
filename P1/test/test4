echo salam > file3
cat file3
wc < file3 > file4
cat file4
rm file3
rm file4
