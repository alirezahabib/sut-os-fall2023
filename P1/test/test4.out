
salam
1 1 6
